src/api/app/views/webui/request/_superseded_by_message.html.haml

```diff
@@ -3,20 +3,17 @@
     .ui-state-info.ui-corner-all.ui-widget-shadow.padding-10pt#request-supersedes-hint
       %span.ui-icon.ui-icon-info
         This request supersedes:
-        - superseding.each do |bs_request|
-          = link_to "request #{bs_request.number}", request_show_path(number: bs_request.number)
-          (
-          = link_to 'Show diff', request_show_path(number: bs_request.superseded_by, diff_to_superseded: bs_request.number)
-          )
+        :ruby
```

Comment:
hpw are lines 6 to 10 related to _Introduce request_and_diff partial_? :confused: